include/rttest/utils.h

```diff
@@ -64,25 +64,24 @@ static inline bool subtract_timespecs(const struct timespec * t1,
   return true;
 }
 
-static inline unsigned long timespec_to_long(const struct timespec * t)
+static inline size_t timespec_to_long(const struct timespec * t)
 {
   return t->tv_sec * NSEC_PER_SEC + t->tv_nsec;
 }
 
-static inline void long_to_timespec(const unsigned long input, struct timespec * t)
+static inline void long_to_timespec(const size_t input, struct timespec * t)
```

Comment:
Not related to the PR but `const` doesn't make much sense on a primitive value passed by value.

Same below.